data/brands/amenity/music_school.json

```diff
@@ -35,7 +35,28 @@
         "amenity": "music_school",
         "brand": "School of Rock",
         "brand:wikidata": "Q7756376",
-        "name": "School of Rock"
+        "brand:zh": "搖滾教室",
+        "name": "School of Rock",
+        "name:zh": "搖滾教室"
+      }
+    },
+    {
+      "displayName": "Yamaha音樂教室",
+      "id": "yamahamusicschool-fc8739",
+      "locationSet": {"include": ["tw"]},
+      "tags": {
+        "amenity": "music_school",
+        "brand": "ヤマハ音楽教室",
+        "brand:en": "Yamaha Music School",
+        "brand:ja": "ヤマハ音楽教室",
+        "brand:ja-Latn": "Yamaha Ongaku Kyōshitsu",
```

Comment:
Is it necessary to use `ja-Latn` in Taiwan? I know Yamaha is from Japan so `ja` is reasonable.